docs/content/guide/echo/README.md

```diff
@@ -49,11 +49,17 @@ If you're using `react`, DXOS provides a simple [UI flow](./react/#joining-space
 
 ## Objects
 
-Units of data are referred to as `objects` (like documents or rows in other databases). `Objects` always belong to a space. Objects can have fields with values, and weak references to other objects to form trees or graphs.
+Units of data are referred to as `objects` (like documents or rows in other databases). `Objects` always belong to a space. `Objects` can have fields with values, and weak references to other objects to form trees or graphs.
+
+## Values
+
+Values within an `object` are JS strings, numbers, booleans, and null, as well as objects and arrays of the same.
```

Comment:
The [linked issue](https://github.com/dxos/dxos/issues/4338) mentions dates as being allowed. However going by [this discord message](https://discord.com/channels/837138313172353095/837688677608980521/1240002548102987928) and a quick check it seems like that's no longer current, so I left dates out.